src/app/components/servers/servers.component.ts

```diff
@@ -98,10 +98,12 @@ export class ServersComponent implements OnInit, OnDestroy {
   }
 
   async startServer(server: Server) {
+    server.status = "starting";
```

Comment:
Shouldn't we rely on serverManagement service?

Reply:
Well, server is not yet running but we sent request. I dunno what you mean by that, we rely on serverManagementService

Reply:
await this.serverManagement.start(server); should update.status to "starting" and propagate changes to UI.

Reply:
The status will be then changed to 'started'

Reply:
As I understand introducing 'starting' status is not desirable

Reply:
It is, but maybe in serverManagement service, not in ServersComponent

Reply:
Done